tgui/packages/tgui/hotkeys.js

```diff
@@ -100,6 +100,39 @@ const getKeyData = (e) => {
   };
 };
 
+const keyCodeToByond = keyCode => {
+  const dict = {16 : 'Shift',
+    17 : 'Ctrl',
```

Comment:
```suggestion
  const dict = {
    16 : 'Shift',
    17 : 'Ctrl',
```

You pressed accept as I was editing lol